docs/data/material/getting-started/support/support.md

```diff
@@ -59,19 +59,23 @@ Receive the latest news on MUI.
 
 Find details on the [supported versions](/versions/#supported-versions).
 
-## Professional support (premium)
+## Technical support (premium)
```

Comment:
How should we name this one?
@mbrookes @oliviertassinari 

```suggestion
## Professional technical support
```

Reply:
> it's understandable that some users create a connection with MUI Core.

I think that it depends on the scope we want for this page, so far, it's not meant to be about MUI Core, but MUI.